packages/getMyPrescriptions/tests/statusUpdate.test.ts

```diff
@@ -48,6 +48,37 @@ describe("Unit tests for statusUpdate", () => {
     const result = buildStatusUpdateData(bundle)
     expect(result).toEqual([])
   })
+
+  test("excludes prescriptions where all items have a status of either 'Prescriber Approved' or 'Cancelled'", async () => {
+    const bundle = mockInteractionResponseBody as Bundle
+
+    bundle.entry?.forEach((entry, index) => {
```

Comment:
Here, you can just add one extension to remove prescription 16B2E0-A83008-81C13H from the status updates. Do bundle.entry[2].resource as Bundle, and then entry[0].resource as MedicationRequest. No need to go through all and add multiple extensions.

Reply:
The test has been updated.
```
    const collectionBundle = bundle.entry![2].resource as Bundle
    const medicationRequest = collectionBundle.entry![0].resource as MedicationRequest
    medicationRequest.extension = [
      {
        url: "https://fhir.nhs.uk/StructureDefinition/Extension-DM-PrescriptionStatusHistory",
        extension: [
          {
            url: "status",
            valueCoding: {code: "Prescriber Approved"}
          },
          {
            url: "statusDate",
            valueDateTime: new Date().toISOString()
          }
        ]
      }
    ]
```